docs/userguide.md

```diff
@@ -372,18 +297,25 @@ for c in cluster1 cluster2; do
     echo; echo
 done
 ```
-
-If you were able to verify the resources removed and added back then you have
+If you were able to verify that the resources were removed and re-added, you have
 successfully verified a working federation-v2 deployment.
 
-#### Using Cluster Selector
+#### Cleaning up
+
+To cleanup the example simply delete the namespace:
```

Comment:
Please add something like 'deletion of the test namespace may be delayed while federation controllers perform the pre-deletion step of removing managed resources from member clusters'.

Reply:
Done now, see NOTE.